feedback/email_server.py

```diff
@@ -121,7 +85,7 @@ def send_email_v3(
         send_email_via_ses(email_message=email_message)
```

Comment:
why do we want to suppress any send errors?
```suggestion
    try:
        send_email_via_ses(email_message=email_message)
    except ClientError:
        logger.error("Failed to send email via SES.")
```

Reply:
We already log out within the function, no need for us to record another log which contains less information than the 1st